examples/3d/spotlight.rs

```diff
@@ -64,6 +72,8 @@ fn setup(
         emissive: Color::rgba_linear(50.0, 0.0, 0.0, 0.0),
         ..default()
     });
+
+    // TODO: Batch spawn
```

Comment:
boop

Reply:
Thanks for catching this! Due to https://github.com/bevyengine/bevy/pull/11981#issuecomment-1958577613, I think I'll likely leave it as a `for` loop.